### DISCUSSION TITLE
简繁体切换

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2517] [插件说明]
描述
---

> 进入阅读模式后，切换简体 / 繁体。

地址
---

[https://simpread.ksria.cn/plugins/details/0sZZtfBYyU](https://simpread.ksria.cn/plugins/details/0sZZtfBYyU)

功能
---

1. 支持快捷键 <kbd>t d</kbd>
2. 可在阅读模式 → 右下角 → 控制栏 → 动作 → 插件触发器  → 切换简体 / 繁体

选项
---

![image](https://user-images.githubusercontent.com/81074/125182479-e30d0e00-e240-11eb-8315-36c7072c71ea.png)

![image](https://user-images.githubusercontent.com/81074/125182620-2451ed80-e242-11eb-8054-595140ec45e7.png)

更新日志
---

### 0.0.1

- 支持快捷 `t d` 触发
- 设置
    - 进入阅读模式后，可选：`默认` `简体` `繁体`

注意
---

> 因标注会截取原始字符，所以在简体状态下标注后，再切换到繁体状态，则无法显示标注位置。（繁体状态同理）

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by RoyLi-Pvt** [comment]
仔细测试了下发现转换不全
转换前:在被心神不寧狀態極差的小胖子連坑四把後
转换后:在被心神不寧状態极差的小胖子连坑四把后
预期的结果:在被心神不宁状态极差的小胖子连坑四把后
@Kenshin

**Reply by Kenshin** [reply]
收到，后续看下，这个转换使用了第三方的词库，可能是词库不全造成的。

**Reply by sche11** [reply]
> 收到，后续看下，这个转换使用了第三方的词库，可能是词库不全造成的。

我在使用这个脚本[去繁留简-繁简转换](https://greasyfork.org/zh-CN/scripts/378243-%E5%8E%BB%E7%B9%81%E7%95%99%E7%AE%80-%E7%B9%81%E7%AE%80%E8%BD%AC%E6%8D%A2),看了下代码,是基于本地词库替换的,绝大部分都可以覆盖到,在日常使用中只有极个别没有正确转换.可以把简悦的繁简转换插件的第三方词库换成这个.

**Comment by YZDNFJ** [comment]
如果可以自行引入第三方字库就太好了，这对于文字语言文化研究工作有非常大的帮助。您方便介绍一下使用的第三方字库吗？

**Reply by Kenshin** [reply]
哈，第三方字库跟切换功能无关。你可以使用 https://github.com/Kenshin/simpread/discussions?discussions_q=is%3Aopen+label%3Atheme 里面提到的一些方式实现。
